scala/src/main/org/apache/spark/api/csharp/CSharpRDD.scala

```diff
@@ -65,7 +65,7 @@ class CSharpRDD(
       command(i) = bytes(i)
     }
 
-    if (CSharpRDD.maxCSharpWorkerProcessCount >= 0) {
```

Comment:
No. In the original design, we think there is no concurrent job running in the application. But pre-fetch changes this assumption. So this changes the implementation of long running C# process..